packages/material-ui-lab/src/SpeedDialAction/SpeedDialAction.js

```diff
@@ -62,7 +58,6 @@ class SpeedDialAction extends React.Component {
     return (
       <Tooltip
         id={id}
-        className={classNames(classes.root, classNameProp)}
```

Comment:
If `className` is no longer applied in Tooltip, that sounds like a bug introduced in the Tooltip refactor, even if the `root` style is now redundant.

Reply:
This was an internal property from the [first iteration](https://github.com/oliviertassinari/material-ui/blame/c46c73424c53c003b2b80e5269fc0cd7f793651e/src/Tooltip/Tooltip.js) onwards. It was applied to the `Popper` element and broke during #9453.

Reply:
Not internal, we just don't document default React properties / HTML attributes.

Reply:
I don't understand why a prop-type validator has to be set for this property if it should not be documented.

So the fault here is with `Tooltip`. Do you want me to open a separate PR that applies a `className` to the `Popper` component or include this here?